update UG

BoAi01 opened the pull request:
* adjust format
* delete check command
* update examples to use 1-based index

cetigerlily commented:
Thank you for the changes! The only thing I'm a bit confused about - I thought we aren't deleting the link location command description in the UG even though we are "removing it", so how come we are deleting the check command description?

BoAi01 commented:
Correct me if I am wrong: I thought we have deleted the `check` command in our app, as we no longer display `availability`to the users (because ensuring the correctness of availability is error-prone). Since we no longer have the `check` command, I removed it from the UG.

BoAi01 commented:
@wxxedu can you take a look at this issue?

cetigerlily reviewed (approved):
LGTM!
